administration/src/components/verification/VerificationController.tsx

```diff
@@ -0,0 +1,110 @@
+import { Button, Card, H1, Spinner } from '@blueprintjs/core'
+import React, { useCallback, useContext } from 'react'
+import { useLocation, useParams } from 'react-router-dom'
+import styled from 'styled-components'
+import JsonFieldView from '../applications/JsonFieldView'
+import ErrorHandler from '../../ErrorHandler'
+import { useGetApplicationQuery, useVerifyApplicationVerificationMutation } from '../../generated/graphql'
+import { ProjectConfigContext } from '../../project-configs/ProjectConfigContext'
+import NotFoundView from '../NotFoundView'
+import { useAppToaster } from '../AppToaster'
+
+const CARD_PADDING = 20
+
+const ApplicationViewCard = styled(Card)`
+  width: 800px;
+  margin: 10px;
+`
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  justify-content: center;
+  padding: ${CARD_PADDING}px;
+`
+
+const ButtonContainer = styled.div`
+  display: flex;
+  width: inherit;
+  flex-direction: row;
+  justify-content: space-around;
+`
+
+type ApplicationVerificationProps = {
+  applicationVerificationId: number
+  applicationAccessKey: string
+}
+
+const ApplicationVerification = ({ applicationVerificationId, applicationAccessKey }: ApplicationVerificationProps) => {
+  const config = useContext(ProjectConfigContext)
+  const appToaster = useAppToaster()
+  const showErrorToaster = () => appToaster?.show({ intent: 'danger', message: 'Etwas ist schief gelaufen.' })
+  const [verifyApplicationVerification, { loading: verificationLoading }] = useVerifyApplicationVerificationMutation({
+    onError: error => {
+      console.error(error)
+      showErrorToaster()
+    },
+  })
+
+  const submitApplicationVerification = useCallback(
+    (verified: boolean) => {
+      verifyApplicationVerification({
+        variables: {
+          applicationVerificationId,
+          verified,
+        },
+      })
+    },
+    [applicationVerificationId, verifyApplicationVerification]
+  )
+
+  const { loading, error, data, refetch } = useGetApplicationQuery({
+    variables: { applicationVerificationId, accessKey: applicationAccessKey },
+  })
+
+  if (loading) return <Spinner />
+  else if (error || !data) return <ErrorHandler refetch={refetch} />
+
+  const { jsonValue, id, verifications } = data.application
+  const verification = verifications.length >= 0 ? verifications[0] : null
+  const jsonField = JSON.parse(jsonValue)
+  const baseUrl = `${process.env.REACT_APP_API_BASE_URL}/application/${config.projectId}/${id}`
+  return (
+    <Container>
+      <H1>{config.name}</H1>
+      <span>
+        Können Sie die Angaben, die die Organisation <b>{verification?.organizationName}</b> betreffen bestätigen?
```

Comment:
should there be a check
```js
if(!verification) {
return null
}
```
I mean if there is no organization to verify who should verify then? ;)

Reply:
The verification should always exist, as otherwise the application could not be loaded. However I could still add the check if you insist :)